### DISCUSSION TITLE
Icon: hexagon vs septagon

### DISCUSSION THREAD

**Discussion by rdwz** [podman-desktop/podman-desktop#734] [Design Feedback]
I was wondering why this design decision was made. All poman related projects and the containers-logo showing septagons, only podman-desktop stands out.

![podman-desktop-logo](https://user-images.githubusercontent.com/8125196/198828805-65847899-3de2-4d6f-9303-05e851c7bae1.png)

![containers-full-horiz](https://user-images.githubusercontent.com/8125196/198828644-1838e19a-d66c-4253-be26-62bb168adcee.png)
![podman(1)](https://user-images.githubusercontent.com/8125196/198828710-babba7e9-47ed-4020-9ec8-31e71c687049.svg)
![68747470733a2f2f63646e2e7261776769742e636f6d2f636f6e7461696e6572732f6275696c6461682f6d61696e2f6c6f676f732f6275696c6461682d6c6f676f5f6c617267652e706e67](https://user-images.githubusercontent.com/8125196/198828754-c2d22b29-b092-4d0b-937f-a537f98f4318.png)
![68747470733a2f2f63646e2e7261776769742e636f6d2f636f6e7461696e6572732f736b6f70656f2f6d61737465722f646f63732f736b6f70656f2e737667](https://user-images.githubusercontent.com/8125196/198828755-93d759bc-c3f6-431a-8d9c-864a5fbeb618.svg)
![crun](https://user-images.githubusercontent.com/8125196/198828643-b69051c8-612e-4bf8-9b14-f23f59ace103.svg)

**Comment by Luxvao** [comment]
I only noticed this after you pointed it out, but ngl I kinda like the hexagon logo

**Comment by slemeur** [comment]
We initially started the design of the icon with the generic "container" shape in mind. 

<img width="382" alt="Capture d’écran 2022-11-02 à 10 31 24" src="https://user-images.githubusercontent.com/1636769/199454973-bc4859ef-dc64-44f9-a20c-a255888c2547.png">

It looks simpler than a septagon, even if I agree it's not very consistent. 
<img width="339" alt="Capture d’écran 2022-11-02 à 10 46 23" src="https://user-images.githubusercontent.com/1636769/199457927-7e0e0b60-3576-46be-bfcc-dd6a5ef6c2b4.png">


If you have other suggestions, we are completely opened to it.

**Comment by cdrage** [comment]
👍 On Septagon. Is more consistent with other projects.

**Comment by rujutashinde** [comment]
closing this discussion as it is either implemented or not needed
